src/api/event.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+import { getEventsByCreationNumber } from "../internal/event";
+import { AnyNumber, GetEventsByCreationNumberResponse } from "../types";
+import { AptosConfig } from "./aptos_config";
+
+/**
+ * A class to query all `Event` Aptos related queries
+ */
+export class Event {
+  readonly config: AptosConfig;
+
+  constructor(config: AptosConfig) {
+    this.config = config;
+  }
+
+  /**
+   * Get events by creation number and the address
+   *
+   * @param args.address - The account address
```

Comment:
i think for param docstrings we've been omitted args

Reply:
I think that was more just doing it quick.  We should include args / handle the docs for all the stuff prior to alpha